vcd_cli/cluster.py

```diff
@@ -88,21 +88,43 @@ def list(ctx):
               default=2,
               metavar='<nodes>',
               help='Number of nodes to create')
+@click.option('-c',
+              '--cpu',
+              'cpu_count',
+              required=False,
+              default=None,
+              metavar='<cpu-count>',
+              help='Number of virtual cpus on each node')
+@click.option('-m',
+              '--memory',
+              'memory',
+              required=False,
+              default=None,
+              metavar='<memory>',
+              help='Amount of memory (in MB) on each node')
 @click.option('-n',
               '--network',
               'network_name',
               default=None,
               required=False,
               metavar='<network>',
               help='Network name')
-@click.option('storage_profile',
-              '-s',
+@click.option('-s',
               '--storage-profile',
+              'storage_profile',
               required=False,
               default=None,
               metavar='<storage-profile>',
               help='Name of the storage profile for the nodes')
-def create(ctx, name, node_count, network_name, storage_profile):
+@click.option('-k',
+              '--ssh_key',
+              'ssh_key',
+              required=False,
+              default=None,
+              metavar='<ssh_key>',
+              help='SSH key to talk to the guest os on the vm')
```

Comment:
The `ssh-key` should be the name of a file where the public ssh key is stored. the command should read the content of the file and pass it to the `create` method. click file option should be used.

Reply:
Addressed.